src/Workspaces/Core/Portable/Indentation/AbstractIndentationService.cs

```diff
@@ -30,28 +29,15 @@ private IEnumerable<AbstractFormattingRule> GetFormattingRules(Document document
             return formattingRules;
         }
 
-        public virtual IndentationResult? GetDesiredIndentation(Document document, int lineNumber, CancellationToken cancellationToken)
+        public IndentationResult GetDesiredIndentation(Document document, int lineNumber, CancellationToken cancellationToken)
```

Comment:
can no longer fail.

Reply:
TryGetSmartTokenIndentation can return true with default.. so, it return default rather than null since Indetationresult is struct.

is that expected behavior?

Reply:
will fix.

Reply:
Fixed.